Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.py

```diff
@@ -5404,7 +5417,7 @@ def main():  # pragma: no cover
         elif command == "nexpose-delete-vulnerability-exception":
             results = delete_vulnerability_exception_command(client=client, vulnerability_exception_id=args.pop("id"))
         elif command == "nexpose-delete-site":
-            results = delete_site_command(client=client, **args)
+            results = delete_site_command(client=client, site_id=args.pop("id", None), **args)
```

Comment:
we don't have this command in the test playbook? how didn't it fail there?

Reply:
The error is `delete_site_command() received an unexpected keyword arg 'id'`, which happened because we passed the`id` key (because of `**args`) as it's on the YAML file instead of the correct `site_id` key.

There is a test for this command, but at the function level (same for all other tests), so it doesn't use `demisto.args`, we just pass it the parameters.

The test looks like this:
```python
@pytest.mark.parametrize("site_id",
                         [
                             ("1",),
                         ])
def test_delete_site_command(mocker, mock_client: Client, site_id: str):
    http_request = mocker.patch.object(BaseClient, "_http_request", return_value={})
    result = delete_site_command(client=mock_client, site_id=site_id)

    http_request.assert_called_with(
        url_suffix=f"/sites/{site_id}",
        method="DELETE",
        resp_type="json",
    )

    assert result.outputs is None
```

Reply:
because of it I asked about teat **playbook** and not unit test :)